### PR TITLE
9.3 streamline versions

### PR DESCRIPTION
**Pull request by weissreto**

### CODE REVIEW

**Comment by weissreto** [comment]
Same changes as on master. Not all changes can be applied since 9.3.1 plugin and 9.3.0 web-tester has some restrictions.

File: deploy/pom.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>ch.ivyteam.ivy.project.demo.ci.deploy</groupId>
+  <artifactId>deploy.modules</artifactId>
+  <version>9.3.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>application</module>
+<!--    <module>single-project</module> -->
```
Comment (ivy-lli) [inline deploy/pom.xml:10]: still needed?

File: Jenkinsfile [hunk 2]
```diff
@@ -38,6 +38,17 @@ pipeline {
...
+          }
+          docker.build('maven').inside() {
+                maven cmd: "-f deploy/single-project/pom.xml clean install " +
+                  "-Divy.engine.list.url=${params.engineListUrl} " +
+                  "-Divy.engine.version=[9.3.0,9.4.0\\) " 
+          }          
+          docker.build('maven').inside() {
+                maven cmd: "-f deploy/single-project-over-http/pom.xml clean install " +
+                  "--settings deploy/single-project-over-http/settings.xml " +
+                  "-Divy.engine.list.url=${params.engineListUrl} " +
```
Comment (ivy-rew) [inline Jenkinsfile:50]: some duplication here: 44+49 ... you may easily share this in a variable:
```
def ivyParams= "-Divy.engine.list.url=${params.engineListUrl} " +
                  "-Divy.engine.version=[9.3.0,9.4.0\\) " 
docker.build('maven').inside() {
                maven cmd: "-f deploy/single-project/pom.xml clean install ${ivyParams}"
```

**Review by ivy-lli — APPROVED** [review]
Looks good

**Review by ivy-rew — APPROVED** [review]
Cool. Thanks for splitting the history

File: deploy/pom.xml (hunk 1 above)
Comment (weissreto) [reply deploy/pom.xml:10]: Yes, to see that this is missing and not correct on this branch. I had to implement a work around to get this stuff built. See Jenkinsfile where the addional projects are explicit built.

**Comment by weissreto** [comment]
Sorry guys. I had to redo this branch because I rebased it on a wrong commit. However the changes should be the same see https://github.com/axonivy/project-build-examples/pull/66.

File: Jenkinsfile (hunk 2 above)
Comment (weissreto) [reply Jenkinsfile:50]: I will do on the new PR.

**Pull request closed without merging**